### PR TITLE
Added validations

### PR DESCRIPTION
**Pull request by lacasera**

### CODE REVIEW

File: src/MapImportedContent.php [hunk 1]
```diff
@@ -170,16 +185,67 @@ protected function castAttribute(string $column, string $attribute, string $mode
...
         }
 
         return $value;
     }
 
+    protected function validateAttribute(string $column, string $attribute, string $model, array $row): ?bool
+    {
+        $value = array_key_exists($column, $row) ? $row[$column] : null;
+
+        if (!$this->validators) {
```
Comment (simoebenhida) [inline src/MapImportedContent.php:199]: It looks the casting will not happen if validation array is empty? This is what you want?

File: src/MapImportedContent.php [hunk 2]
```diff
@@ -170,16 +185,67 @@ protected function castAttribute(string $column, string $attribute, string $mode
...
+            return true;
+        }
+
+        $validation = $this->validators->filter(function ($value, $key) use ($model) {
+            return $key === $model;
+        });
+
+        if (!$validation) {
```
Comment (simoebenhida) [inline src/MapImportedContent.php:207]: Same here

File: src/MapImportedContent.php [hunk 3]
```diff
@@ -170,16 +185,67 @@ protected function castAttribute(string $column, string $attribute, string $mode
...
+            return true;
+        }
+
+        $modelValidations = Arr::get($validation, $model, []);
+
+        if (array_key_exists($attribute, $modelValidations)) {
```
Comment (simoebenhida) [inline src/MapImportedContent.php:213]: Better to wrap the validation alone and casting alone

Comment (lacasera) [reply src/MapImportedContent.php:214]: done @simoebenhida

**Pull request merged**